beacon-chain/db/filesystem/blob.go

```diff
@@ -234,6 +240,12 @@ func (bs *BlobStorage) Prune(currentSlot primitives.Slot) error {
 			if err := bs.processFolder(folder, currentSlot, retentionSlots); err != nil {
 				return err
 			}
+			num, err := bs.countFiles(folder.Name())
+			if err != nil {
+				return err
+			}
+			blobsPrunedCounter.Add(float64(num))
```

Comment:
This doesn't look right. Pruned counter shouldn't be increased by the number of files on disk, right? Perhaps you want `processFolder` to return `(pruned uint, error)`